services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/pdf/MapBounds.kt

```diff
@@ -0,0 +1,60 @@
+package fi.hel.haitaton.hanke.pdf
+
+import java.util.Locale
+import kotlin.math.max
+import org.geotools.api.referencing.crs.CoordinateReferenceSystem
+import org.geotools.geometry.jts.ReferencedEnvelope
+import org.geotools.referencing.CRS
+
+data class Point(val x: Double, val y: Double) {
+
+    private fun fixedPoint(a: Double): String = String.format(Locale.UK, "%.3f", a)
+
+    override fun toString(): String = "(x=${fixedPoint(x)}, y=${fixedPoint(y)})"
+}
+
+data class MapBounds(val min: Point, val max: Point) {
+    val xSize: Double = max.x - min.x
+    val ySize: Double = max.y - min.y
+    val center: Point = Point(y = (min.y + max.y) / 2.0, x = (min.x + max.x) / 2.0)
```

Comment:
A suggestion:
```
fun Pair<Point, Point>.center(): Point = Point(
    x = (first.x + second.x) / 2,
    y = (first.y + second.y) / 2,
)
```

and then 
`val center: Point = Pair(min, max).center()`

Reply:
What would be the motivation for this change? This would create a Pair object and instantly throw it away, so there needs to be some benefit to offset the cost.

Reply:
Maybe you're right and it does not bring enough benefits. If you want to extract the calculation logic it could be done in a function instead of Pair.

Reply:
I extracted a function for calculating the center point.